AUTHORS

```diff
@@ -3310,5 +3310,17 @@ Mestery <mestery@pm.me>
 Himadri Ganguly <himadri.tech@gmail.com>
 Howie Zhao <howiezhaohr@hotmail.com>
 Luan Devecchi <luan@engineer.com>
+ashish maurya <ashishmaurya@outlook.com>
+Justin <justinleeong@gmail.com>
+Raz Luvaton <rluvaton@gmail.com>
+Don Jayamanne <don.jayamanne@yahoo.com>
+Felix Yan <felixonmars@archlinux.org>
+Ray <ray@isrc.iscas.ac.cn>
+Node.js GitHub Bot <github-bot@iojs.org>
+Alexey Ten <alexeyten@gmail.com>
+JckXia <ys3xia@uwaterloo.ca>
+Christian Boehlke <christian@boehlkes.de>
+null <60427892+vierofernando@users.noreply.github.com>
```

Comment:
```suggestion
```

Reply:
I get why you might want that removed, but I think this is fine. There are 58 entries already in AUTHORS with email addresses of that format. I believe the email addresses of this format are unique identifiers tied to GitHub accounts that do not expose a usable email address. It still represents an author, even if they sometimes choose to be known as "null" or "c0b".  Moreover, the AUTHORS file isn't really much of a useful file and I'd rather expend effort automating and tolerating things like this than coding up exception cases for a file no one reads or uses. If we remove this manually, it will just appear the next time this runs.

<details>

<summary>The 58 current entries that are in this format</summary>

```console
$ egrep '<\d+\+[^@]+@users.noreply.github.com>' AUTHORS
Alex Gresnel <31708810+agresnel@users.noreply.github.com>
c0b <14798161+c0b@users.noreply.github.com>
cPhost <23620441+cPhost@users.noreply.github.com>
gowpen <33104741+gowpen@users.noreply.github.com>
Cyril Lakech <1169286+clakech@users.noreply.github.com>
BufoViridis <30531954+BufoViridis@users.noreply.github.com>
nodeav <30617226+nodeav@users.noreply.github.com>
Petras <15868923+kimberlake@users.noreply.github.com>
Kevin Lacabane <5239883+klacabane@users.noreply.github.com>
prayag21 <10997858+prayag21@users.noreply.github.com>
Brandon Lee <40652534+brandonlwt@users.noreply.github.com>
Eugen Cazacu <32613393+oygen87@users.noreply.github.com>
James Bromwell <943160+thw0rted@users.noreply.github.com>
Lovingly <42682205+lovinglyy@users.noreply.github.com>
siddhant <30566406+siddhant1@users.noreply.github.com>
Murtaza H <1850989+mhussa19@users.noreply.github.com>
Jackson Chui <14085209+haiXchuus@users.noreply.github.com>
Charalampos Fanoulis <38255093+cfanoulis@users.noreply.github.com>
Gerhard Stoebich <18708370+Flarna@users.noreply.github.com>
Karen He <32376376+baekrxnn@users.noreply.github.com>
PaulBags <19583196+PaulBags@users.noreply.github.com>
Marian Rusnak <4215517+marian-r@users.noreply.github.com>
Unlocked <10186337+TheUnlocked@users.noreply.github.com>
Nicolas Thumann <46975855+n-thumann@users.noreply.github.com>
Soar <58918356+soar-beep@users.noreply.github.com>
Colin Grant <43424668+GKJCJG@users.noreply.github.com>
SpaceRacet5w2A6l0I <59487396+SpaceRacet5w2A6l0I@users.noreply.github.com>
kenzo-spaulding <51864231+kenzo-spaulding@users.noreply.github.com>
ExE Boss <3889017+ExE-Boss@users.noreply.github.com>
Moni <40552237+NotMoni@users.noreply.github.com>
William Armiros <54150514+willarmiros@users.noreply.github.com>
Ishaan Jain <37652866+ishaanjain1898@users.noreply.github.com>
Deep310 <55121371+Deep310@users.noreply.github.com>
WilliamConnatser <43946230+WilliamConnatser@users.noreply.github.com>
falguniraina <48027052+falguniraina@users.noreply.github.com>
Shigma <33423008+Shigma@users.noreply.github.com>
schamberg97 <50446906+schamberg97@users.noreply.github.com>
ax1 <16510021+ax1@users.noreply.github.com>
Shubham Parihar <51517103+iShibi@users.noreply.github.com>
Ben Turner <7623873+ben-turner@users.noreply.github.com>
masx200 <34191203+masx200@users.noreply.github.com>
Brian Ingenito <28159742+bingenito@users.noreply.github.com>
inokawa <48897392+inokawa@users.noreply.github.com>
Michael Chen <4326639+mcgitty@users.noreply.github.com>
Mattia Pontonio <44380480+mattiapontonio@users.noreply.github.com>
Akash Negi <55234838+NegiAkash890@users.noreply.github.com>
Jay Tailor <60511316+JayvaScript@users.noreply.github.com>
humanwebpl <58517331+humanwebpl@users.noreply.github.com>
Merlin Luntke <22600241+Luntke@users.noreply.github.com>
TodorTotev <51530311+TodorTotev@users.noreply.github.com>
yotamselementor <83912471+yotamselementor@users.noreply.github.com>
julianjany <54538266+julianjany@users.noreply.github.com>
Derevianchenko Maksym <32910350+maks-white@users.noreply.github.com>
RA80533 <32469082+RA80533@users.noreply.github.com>
FrankEntriken <42781627+FrankEntriken@users.noreply.github.com>
Jacob <3012099+JakobJingleheimer@users.noreply.github.com>
ejose19 <8742215+ejose19@users.noreply.github.com>
Nikita Rykov <40735471+angrymouse@users.noreply.github.com>
$ 
```

</details>

Reply:
Optimistically marking as resolved. Please undo my resolution mark if that's wrong.